src/pkcs15init/pkcs15-westcos.c

```diff
@@ -246,7 +246,12 @@ static int westcos_pkcs15init_generate_key(sc_profile_t *profile,
 	if (BN_set_word(bn, RSA_F4) != 1 ||
 	    EVP_PKEY_keygen_init(pctx) != 1 ||
 	    EVP_PKEY_CTX_set_rsa_keygen_bits(pctx, key_info->modulus_length) != 1 ||
-	    EVP_PKEY_CTX_set1_rsa_keygen_pubexp(pctx, bn) != 1 ||
+#if OPENSSL_VERSION_NUMBER >= 0x30000000L
+		EVP_PKEY_CTX_set1_rsa_keygen_pubexp(pctx, bn) != 1 ||
```

Comment:
```suggestion
	    EVP_PKEY_CTX_set1_rsa_keygen_pubexp(pctx, bn) != 1 ||
```
please, indent the second part with spaces to keep the alignment with the rest of the code regardless the tab width

Reply:
OK